src/GrassPokemon.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class GrassPokemon extends Pokemon {
+
+    private static final String type = "gp"; // not quite sure, if this will work here ?? //
```

Comment:
Nee, dit werkt niet. In de PokemonGymImpl class wordt in een switch-statement gechecked of het type van de pokemon gelijk is aan "grass".